content/00.build/65.developer-reference/50.era-contracts/25.l1-ecosystem-contracts.md

```diff
@@ -0,0 +1,689 @@
+---
+title: L1 Ecosystem Contracts
+description:
+---
+
+Ethereum's future is rollup-centric.
+This means breaking with the current paradigm of isolated EVM chains to infrastructure
+that is focused on an ecosystem of interconnected zkEVMs/zkVMs, (which we name Hyperchains).
+This ecosystem will be grounded on Ethereum, requiring the appropriate L1 smart contracts.
+Here we outline our ZK Stack approach for these contracts, their interfaces,
+the needed changes to the existing architecture, as well as future features to be implemented.
+
+If you want to know more about Hyperchains, check this blogpost on an
+[Introduction to hyperchains](https://blog.matter-labs.io/introduction-to-hyperchains-fdb33414ead7),
+or go through our docs on [hyperscaling](https://era.zksync.io/docs/reference/concepts/hyperscaling.html).
+<!-- TODO: no page on hyperscaling -->
```

Comment:
This hyperscaling page doesn't exist anymore on zksync.io , we should instead use `/zk-stack/concepts/zk-chains`

Reply:
I agree, since both the docs are same, we can just keep one.